es6/tests/base.js

```diff
@@ -916,3 +920,65 @@ describe("Serialization", function() {
 		JSON.stringify(doc);
 	});
 });
+
+describe("Docxtemplater v4 tests", function() {
+	it("should work with new constructor", function() {
+		let xmlDocuments;
+
+		const module = {
+			requiredAPIVersion: "3.0.0",
+			optionsTransformer(options, docxtemplater) {
+				const relsFiles = docxtemplater.zip
+					.file(/document.xml.rels/)
+					.map(file => file.name);
+				options.xmlFileNames = options.xmlFileNames.concat(relsFiles);
+				return options;
+			},
+			set(options) {
+				if (options.xmlDocuments) {
+					xmlDocuments = options.xmlDocuments;
+				}
+			},
+		};
+
+		const moduleV4Templater = {
```

Comment:
I think you can remove this module, and use it only on a v4 constructed docxtemplater